index.html

```diff
@@ -976,8 +976,8 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
         </li>
         <li>
           If |this|.|data| is not a {{ReadableStream}} or if
-          |dataUsed| is `true`, or if |form| is not an {{object}} or if |schema| or its
-          |type| are `null` or `undefined`, then
+          |dataUsed| is `true`, or if |form| is not an {{object}} or if |schema|
+          is `null` or `undefined`, then
```

Comment:
For reference, this is (more or less) the change proposed by @relu91 in https://github.com/w3c/wot-scripting-api/pull/534#pullrequestreview-1838591689. The question here, also raised by him, is how we should ensure that the schema is valid – do we need to define a separate algorithm for that?

```suggestion
          |dataUsed| is `true`, or if |form| is not an {{object}} or if |schema|
          is `null` or `undefined` or not a valid schema, then
```

Reply:
Oh, yeah, I now remember that we also discussed this briefly on Friday during the Thingweb meeting... Then I suppose we should keep this step as is, right?

Reply:
"not a valid schema" needs an algorithm